assets/js/jquery-tiptip/jquery.tipTip.js

```diff
@@ -58,40 +58,40 @@
 			}
 			if(org_title != ""){
 				if(!opts.content){
-					org_elem.removeAttr(opts.attribute); //remove original Attribute
+					org_elem.prop(opts.attribute, false); //remove original Attribute
```

Comment:
@roykho I think this is breaking some extensions using `jquery-tiptip` (see 693-gh-woocommerce/woocommerce-product-addonsr example).